app/javascript/styles/mastodon/components.scss

```
@@ -5798,6 +5798,7 @@ a.status-card.compact:hover {
   &__container {
     padding: 30px;
     pointer-events: all;
+    overflow-x: auto;
```

Shouldn't it be `overflow-y: auto`? I think horizontal overflow should wrap, and vertical overflow is what we need to take care of.
```suggestion
    overflow-y: auto;
```

---

Indeed… thank you for the correction!